sections/advanced/theming.md

```diff
@@ -109,6 +109,19 @@ class MyComponent extends React.Component {
 export default withTheme(MyComponent);
 ```
 
+You can also use `useContext` to access the current theme outside of styled components when working with React Hooks.
+
+```jsx
+import { ThemeContext } from 'styled-components';
```

Comment:
Can you add the useContext import here too?

Reply:
This also requires v4, if you make a subheading for this we have a tagging structure for this (just grep for "v4")

Reply:
Sure thing will edit in a minute here.